rp2040-hal/src/pio.rs

```diff
@@ -360,6 +360,24 @@ impl<P: PIOExt, SM: StateMachineIndex> ValidStateMachine for (P, SM) {
     }
 }
 
+/// Pin State
+///
+/// Note this may not be the same as its electrical state (e.g. if invert is enabled on the pin).
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+#[allow(missing_docs)]
```

Comment:
Why missing docs? Could you just give short doc for enum variants, say "Pin is in high state".

Reply:
It seems redundant to document `PinState::Active` with
> Pin in active state.

But I can of course add those line.